FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_TCP_IP.c

```diff
@@ -967,8 +967,8 @@ static void prvTCPCreateWindow( FreeRTOS_Socket_t *pxSocket )
 	}
 	vTCPWindowCreate(
 		&pxSocket->u.xTCP.xTCPWindow,
-		( ( size_t ) ipconfigTCP_MSS ) * pxSocket->u.xTCP.uxRxWinSize,
-		( ( size_t ) ipconfigTCP_MSS ) * pxSocket->u.xTCP.uxTxWinSize,
+		ipconfigTCP_MSS * pxSocket->u.xTCP.uxRxWinSize,
+		ipconfigTCP_MSS * pxSocket->u.xTCP.uxTxWinSize,
```

Comment:
Would this not affect the functionality?

Reply:
Typecasts _can_ affect the value of a variable, for example if casting to a smaller sized type, between `float` and `int` types, or between signed and unsigned types when dealing with values in which the high bit is set.  But because [we know the range of possible values](https://en.wikipedia.org/wiki/Maximum_transmission_unit) for [`ipconfigTCP_MSS`](https://github.com/FreeRTOS/FreeRTOS/blob/master/FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOSIPConfigDefaults.h#L393) (an integer constant), we know that none of these conditions exist here.